account_invoice_select_for_payment/models/account_move.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    selected_for_payment = fields.Boolean("To Pay")
+
+    def action_toggle_select_for_payment(self):
+        selected = self.filtered(lambda rec: rec.selected_for_payment)
+        unselected = self - selected
+        if selected:
+            selected.write({"selected_for_payment": False})
+        if unselected:
+            unselected.write({"selected_for_payment": True})
+
+    def action_register_payment(self, invoice_ids=None):
```

Comment:
Does not look correct, the base declaration of this function is different...
https://github.com/odoo/odoo/blob/ca9a1157dd47927ae7750e6669b7efac1fdcd629/addons/account/models/account_move.py#L5100